visual_behavior/validation/sdk.py

```diff
@@ -0,0 +1,293 @@
+from allensdk.brain_observatory.behavior.behavior_project_cache import BehaviorProjectCache as bpc
+from allensdk.brain_observatory.behavior.behavior_data_session import BehaviorDataSession
+from allensdk.brain_observatory.behavior.behavior_ophys_session import BehaviorOphysSession
+import traceback
+import allensdk
+import datetime
+import sys
+import platform
+import pandas as pd
+import argparse
+import plotly.graph_objs as go
+import warnings
+from visual_behavior import database as db
+
+
+def get_cache():
+    MANIFEST_PATH = "//allen/programs/braintv/workgroups/nc-ophys/visual_behavior/2020_cache/production_cache/manifest.json"
+    cache = bpc.from_lims(manifest=MANIFEST_PATH)
+    return cache
+
+
+def is_ophys(behavior_session_id):
+    cache = get_cache()
+
+    behavior_session_table = cache.get_behavior_session_table()
+
+    return pd.notnull(behavior_session_table.loc[behavior_session_id]['ophys_session_id'])
+
+
+def bsid_to_oeid(behavior_session_id):
```

Comment:
is there a comparable function in data_access.utilities? If not, this type of thing should live there

Reply:
I moved this function to data_access.utilities.